cwms-data-api/src/test/java/cwms/cda/api/location/kind/GateChangeStressTest.java

```diff
@@ -0,0 +1,250 @@
+                                                               .withTooltip("Other release")
+                                                               .build();
+            GateSetting.Builder settingBuilder = new GateSetting.Builder().withOpening(15.)
+                                                                          .withOpeningParameter("Elev")
+                                                                          .withInvertElevation(5.)
+                                                                          .withOpeningUnits("ft");
+            GateChange.Builder changeBuilder = new GateChange.Builder().withProjectId(PROJECT_ID)
+                                                                       .withDischargeComputationType(compType)
+                                                                       .withReasonType(releaseReason)
+                                                                       .withProtected(false)
+                                                                       .withNewTotalDischargeOverride(1.5)
+                                                                       .withOldTotalDischargeOverride(2.0)
+                                                                       .withDischargeUnits("cfs")
+                                                                       .withPoolElevation(50.)
+                                                                       .withTailwaterElevation(30.)
+                                                                       .withElevationUnits("ft")
+                                                                       .withNotes("Notes");
+            List<List<GateChange>> changes = dateBatches.stream()
+                                                        .map(dates -> buildGateChanges(dates, changeBuilder, settingBuilder,
+                                                                                       outletIds))
+                                                        .collect(Collectors.toList());
+
+            LOGGER.atSevere().log("Storing locations...");
+            allLocs.forEach(loc -> storeLocLogException(context, loc));
+            LOGGER.atSevere().log("Storing outlets...");
+            outlets.forEach(outlet -> storeOutlet(context, outlet));
+            LOGGER.atSevere().log("Storing rating spec id...");
+            createRatingSpecForOutlet(context, outlets.get(0), CONDUIT_GATE_RATING_SPEC_ID);
+
+            LOGGER.atSevere().log("Storing changes...");
+            changes.forEach(changeBatch -> {
+                LOGGER.atSevere().log("Storing " + changeBatch.size() + " changes from " + changeBatch.get(0).getChangeDate() + " to " + changeBatch.get(changeBatch.size() - 1));
+                dao.storeOperationalChanges(changeBatch, true);
+            });
+
+        }, CwmsDataApiSetupCallback.getWebUser());
+    }
+
+    @Test
+    void stress_test() {
```

Comment:
Is there a better word than stress here? When I saw that immediately thought "parallel" not bulk data.

Reply:
should be scaled, sorry

Reply:
or scaling?

Reply:
Honestly that seems even more like something that should be parallel.

But yeah, that seems reasonable. Or at least as reasonable as the english language allows. Perhaps someone else will come up with something.